syntax="proto2";

package aap_protobuf.service.bluetooth;

enum BluetoothMessageId
{
    BLUETOOTH_MESSAGE_PAIRING_REQUEST = 32769;
    BLUETOOTH_MESSAGE_PAIRING_RESPONSE = 32770;
    BLUETOOTH_MESSAGE_AUTHENTICATION_DATA = 32771;
    BLUETOOTH_MESSAGE_AUTHENTICATION_RESULT = 32772;
}


